backend-api/src/functions/asyncTxmaEvent/asyncTxmaEventHandler.ts

```diff
@@ -32,6 +48,26 @@ export async function lambdaHandlerConstructor(
     });
     return badRequestResponse("invalid_request", errorMessage);
   }
+  const { sessionId } = validateRequestBodyResult.value;
+
+  const sessionRegistry = dependencies.getSessionRegistry(
+    config.SESSION_TABLE_NAME,
+  );
+  const getSessionResult = await sessionRegistry.getSession(sessionId);
+  const eventService = dependencies.getEventService(config.TXMA_SQS);
+  const eventData = {
+    sessionId,
+    issuer: config.ISSUER,
+    ipAddress: getIpAddress(event),
+    txmaAuditEncoded: getHeader(event.headers, "Txma-Audit-Encoded"),
+  };
```

Comment:
```suggestion
  const { ipAddress, txmaAuditEncoded } = getAuditHeaders(event);
  const eventData = {
    sessionId,
    issuer: config.ISSUER,
    ipAddress,
    txmaAuditEncoded,
  };
```

I wouldn't commit from here as they'll be import changes

Reply:
Nice! Updated